web-ng/src/app/components/project-header/project-header.component.html

```diff
@@ -13,20 +13,23 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
-<div class="header">
-  <div class="header-item">
+<div fxLayout="row" class="header" fxLayoutAlign="flex-self center">
+  <div>
     <img class="logo" src="/assets/img/logo.png" />
-    <span class="ground app-title">Ground</span>
+    <span class="app-title">Ground</span>
     <span class="header-divider"></span>
     <app-inline-edit-title></app-inline-edit-title>
   </div>
-  <div class="header-item">
+  <div fxFlex></div>
```

Comment:
is there any reason why this div is empty?

Reply:
I was using the empty div to stretch the header to full width, but seems this isn't necessary. Moved `fxFlex` to the first `div` instead.